environments/skeleton/{{cookiecutter.environment}}/terraform/control.userdata.tpl

```diff
@@ -0,0 +1,21 @@
+#cloud-config
+disk_setup:
+  /dev/vdb:
+    table_type: gpt
+    layout: true
+  /dev/vdc:
```

Comment:
How can we control for device names here? `/dev/vda` vs `/dev/sda` etc - these are properties of the SCSI device I think...

Reply:
You can't really. It is supposed to depend on the flags set on the image, so setting `--property hw_scsi_model=virtio-scsi` should I think give you `sd*` although in my test it didn't. For the use of this TF I think it's ok - in CI we control both the device name here and the image tags. If you're using this TF in your own deployment you may need to change it but that goes for anything